src/OrchardCore.Modules/Orchard.Layers/Views/Admin/Index.cshtml

```diff
@@ -45,14 +46,14 @@
                                     {
                                         contentItemSummary.Meta.Add(Html.Raw("<span class=\"badge badge-default\"><i class=\"fa fa-bookmark-o\" aria-hidden=\"true\"></i> " + contentItemSummary.ContentItem.Content.LayerMetadata.Layer + "</span>"));
 
-                                        maxPosition = Math.Max(maxPosition, (double)contentItemSummary.ContentItem.Content.LayerMetadata.Position);
+                                        maxPosition = Math.Max(maxPosition, Convert.ToDouble(contentItemSummary.ContentItem.Content.LayerMetadata.Position.ToString(CultureInfo.InvariantCulture)));
```

Comment:
this change is unnecessary, Position is a double, you don't need to serialize then deserialize it.